libs/community/langchain_community/document_loaders/parsers/doc_intelligence.py

```diff
@@ -71,7 +71,7 @@ def _generate_docs_page(self, result: Any) -> Iterator[Document]:
             yield d
 
     def _generate_docs_single(self, result: Any) -> Iterator[Document]:
-        yield Document(page_content=result.content, metadata={})
+        yield Document(page_content=result.content, metadata=result)
```

Comment:
is `result` always a dict? I see it typed as `Any`.

Reply:
No you are right, it is AnalyzeResult which has an as_dict method. I will adjust accordingly, thanks for pointing out!
![image](https://github.com/user-attachments/assets/06415c60-d740-49db-964d-70aa46a1c137)

Reply:
fixed!

Reply:
@ccurme I kindly ask if there is an update in the review process :)